File: copl/com.elster.jupiter.system/src/main/java/com/elster/jupiter/system/Component.java
/*
 * Copyright (c) 2017 by Honeywell International Inc. All Rights Reserved
 */

package com.elster.jupiter.system;

import aQute.bnd.annotation.ProviderType;

@ProviderType
public interface Component {

    String getSymbolicName();

    String getVersion();

    BundleType getBundleType();
}

File: copl/com.elster.jupiter.extjs/src/main/web/js/ext/src/rtl/draw/Component.js
/*
 * Copyright (c) 2017 by Honeywell International Inc. All Rights Reserved
 */
Ext.define('Ext.rtl.draw.Component', {
    override: 'Ext.draw.Component',
    
    initSurfaceCfg: function(cfg) {
        if (this.getHierarchyState().rtl) {
            cfg.isRtl = true;
        }
    }    
});
